dipy/workflows/denoise.py

```diff
@@ -257,10 +257,9 @@ def run(self, input_files, slice_axis=2, n_points=3, num_threads=1,
             third axis.
         n_points : int, optional
             Number of neighbour points to access local TV (see note).
-        num_threads : int or None, optional
-            Number of threads. Only applies to 3D or 4D `data` arrays. If None
-            then all available threads will be used. Otherwise, must be a
-            positive integer.
+        num_threads : int, optional
+            Number of threads. Only applies to 3D or 4D `data` arrays. Must be
+            a positive integer.
```

Comment:
Has this been agreed upon? Not sure this change is desired: this would mean that there would not be a way to use all available threads from the CLIs unless the users know the limits of their hardware. @skoudoro ?

Reply:
I confirm this change is not desired. Can you put it back as it was @areeshatariq? Thank you

Reply:
Addressed in 4ff61ad